cmd/ipfs/daemon.go

```diff
@@ -261,17 +247,18 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 	}
 
 	// construct fuse mountpoints - if the user provided the --mount flag
-	mount, _, err := req.Option(mountKwd).Bool()
-	if err != nil {
+	if err := mountFuse(req); err != nil {
 		res.SetError(err, cmds.ErrNormal)
 		return
 	}
-	if mount {
-		if err := mountFuse(req); err != nil {
+
+	// repo blockstore GC - if --disable-gc flag is not present
+	go func() {
+		if err := runGC(req, repo, node); err != nil {
 			res.SetError(err, cmds.ErrNormal)
 			return
 		}
-	}
+	}()
```

Comment:
With this, early errors like config errs are to be returned only at the end? Or should the errs be separated into err, gcErrc?